src/quo/components/community/token_gating.cljs

```diff
@@ -10,40 +10,36 @@
 
 (defn token-requirement-list-row
   [tokens padding?]
-  [rn/view
-   {:style (style/token-row padding?)}
-   (doall
-    (map-indexed (fn [token-index token]
-                   (let [{:keys [img-src amount sufficient? purchasable?]} token]
-                     ^{:key token-index}
-                     [rn/view {:style style/token-tag-spacing}
-                      [token-tag/view
-                       {:token-symbol  (:symbol token)
-                        :token-value   amount
-                        :size          :size-24
-                        :options       (cond
-                                         sufficient?  :hold
-                                         purchasable? :add)
-                        :token-img-src img-src}]]))
-                 tokens))])
```

Comment:
Small refactor: `doall` is not needed here since we are not using `deref` (`@`) inside a lazy seq